### PR TITLE
Rough Draft changes for moving forward

### PR DESCRIPTION
**Pull request by Okachobe**

### CODE REVIEW

File: ape/util/get_cur_user_info.php [hunk 1]
```diff
@@ -42,11 +42,11 @@ function getCurUserInfo($isClient)
...
                 $userInfo = array('userId' => $_SESSION['ewuid'], 
-                'userType' => $_SESSION["phpCAS"]["attributes"]["UserType"], 
+                'userType' => $_SESSION["UserType"], 
                 'userSession' => session_id(),
-                'userFname' => $_SESSION["phpCAS"]["attributes"]["FirstName"],
-                'userLname' => $_SESSION["phpCAS"]["attributes"]["LastName"],
-                'userEmail' => $_SESSION["phpCAS"]["attributes"]["Email"] );
+                'userFname' => $_SESSION["FirstName"],
+                'userLname' => $_SESSION["LastName"],
+                'userEmail' => $_SESSION"Email"] );
```
Comment (ashermancinelli) [inline ape/util/get_cur_user_info.php]: Could we document the changes we're making here? I would love a table that lays out the old names for things that maps to the new way. For example, I'd love the table to look something like this:

<table>
<thead>
<td>old value</td>
<td>new value</td>
</thead>
<tr>
<td>$_SESSION["phpCAS"]["attributes"]["FirstName"]</td>
<td>$_SESSION["FirstName"]</td>
</tr>
</table>

Or if you stay consistent we can just all know that the new session variables are the same as the CAS ones, just without the ```["phpCAS"]["attributes"]``` bit.

File: client_secret.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"357634610842-mb2qf2ngkh6ifp519kchkhug7l9pa2a7.apps.googleusercontent.com","project_id":"ewu-ape","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"8D9r4JqcURJNYS_HukECvia4","redirect_uris":["http://ape.ewu.edu"],"javascript_origins":["http://ape.ewu.edu"]}}
```
Comment (ashermancinelli) [inline client_secret.json]: Could you remove this from the pull request? This is sensitive information from my google cloud account that lets us use the API, and I'd rather not let it out for everyone to use 😄

Comment (Okachobe) [reply client_secret.json]: How do i remove that from the pull request?

File: ape/util/get_cur_user_info.php (hunk 1 above)
Comment (Okachobe) [reply ape/util/get_cur_user_info.php]: for the table would it belong in the code/website or like a readme?

**Comment by ashermancinelli** [comment]
So most of this is stuff I already pushed to master tonight, so I'm going to close this. If you want to take a look at my latest commit for guidance moving forward, you can make another PR and I can merge that.

**Pull request closed without merging**